### PR TITLE
branch F40; add F41

### PR DESCRIPTION
**Pull request by achilleas-k**
Here we go again.

### CODE REVIEW

**Comment by achilleas-k** [comment]
Started a snapshot job on the branch: https://github.com/osbuild/rpmrepo/actions/runs/7915290353

**Review by ondrejbudai — APPROVED** [review]

File: repo/el9-x86_64-sap-rhui-azure.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (ondrejbudai) [inline repo/el9-x86_64-sap-rhui-azure.json:1]: Chm, what's this change?

**Review by ondrejbudai — CHANGES_REQUESTED** [review]

File: repo/el9-x86_64-sap-rhui-azure.json (hunk 1 above)
Comment (achilleas-k) [reply repo/el9-x86_64-sap-rhui-azure.json:1]: Excellent question.  I don't know.  I ran the make and checked everything in without looking.  Let me see if it's a local leftover of some sort.

Comment (achilleas-k) [reply repo/el9-x86_64-sap-rhui-azure.json:1]: Nope.  Wasn't me.  It's configured in `repo-definitions.yaml` but missing from `main`.

Comment (achilleas-k) [reply repo/el9-x86_64-sap-rhui-azure.json:1]: v

Comment (achilleas-k) [reply repo/el9-x86_64-sap-rhui-azure.json:1]: Split the commits.

Comment (ondrejbudai) [reply repo/el9-x86_64-sap-rhui-azure.json:1]: Hehe, I'm sorry.

**Pull request merged**